Solutions/WithSecureElementsViaConnector/Data Connectors/WithSecureElementsViaConnector.json

```diff
@@ -10,7 +10,12 @@
       "baseQuery": "CommonSecurityLog\n| where DeviceVendor == \"F-Secure\""
     }
   ],
-  "sampleQueries": [""],
+ "sampleQueries": [
+  {
+    "description": "All logs",
+    "query": "CommonSecurityLog\n            | sort by TimeGenerated"
```

Comment:
Thanks a lot for help!
One question regarding that query:
Do those samples apply only to the one connector? I mean should there be this where clause to match `DeviceVendor` as it is in `"graphQueries"`?

Reply:
@HubertSzymanski5, yes please suggest the appropriate query as needed.

Reply:
So if that's valid I'd go with:
`"CommonSecurityLog\n| where DeviceVendor == \"F-Secure\"\n| sort by TimeGenerated"`

Should commit that and rebuild package?